package.json

```diff
@@ -23,114 +23,113 @@
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^13.1.9",
     "@types/chart.js": "^2.9.31",
-    "@types/classnames": "^2.2.11",
-    "@types/clipboard": "^2.0.1",
-    "@types/jest": "^26.0.20",
-    "@types/lodash": "^4.14.168",
-    "@types/luxon": "^1.25.1",
-    "@types/node": "^14.14.22",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "@types/react-modal": "^3.10.6",
-    "@types/react-text-mask": "^5.4.6",
+    "@types/jest": "^26.0.23",
+    "@types/lodash": "^4.14.170",
+    "@types/luxon": "^1.27.0",
+    "@types/node": "^15.12.2",
+    "@types/react": "^17.0.11",
+    "@types/react-dom": "^17.0.7",
+    "@types/react-modal": "^3.12.0",
+    "@types/react-text-mask": "^5.4.7",
     "@types/yup": "^0.29.11",
-    "@typescript-eslint/eslint-plugin": "^4.14.2",
-    "@typescript-eslint/parser": "^4.14.2",
-    "autoprefixer": "^9",
+    "@typescript-eslint/eslint-plugin": "^4.27.0",
+    "@typescript-eslint/parser": "^4.27.0",
+    "autoprefixer": "^10.2.6",
     "babel-loader": "^8.2.2",
     "babel-plugin-react-require": "^3.1.3",
     "babel-preset-react-app": "^10.0.0",
     "chokidar-cli": "^2.1.0",
-    "concurrently": "^5.3.0",
-    "cpy-cli": "^3.1.1",
-    "css-color-converter": "^2.0.0",
-    "cssnano": "^4.1.10",
-    "eslint-config-airbnb-typescript": "^12.0.0",
+    "concurrently": "^6.2.0",
+    "cssnano": "^5.0.6",
+    "eslint-config-airbnb-typescript": "^12.3.1",
     "eslint-config-react-app": "^6.0.0",
-    "eslint-plugin-mdx": "^1.8.2",
-    "eslint-plugin-prettier": "^3.3.1",
-    "husky": "^4.3.8",
+    "eslint-plugin-mdx": "^1.13.0",
+    "eslint-plugin-prettier": "^3.4.0",
+    "husky": "^6.0.0",
     "jest-css-modules": "^2.1.0",
-    "postcss": "^7",
-    "postcss-cli": "^7.1.1",
+    "postcss": "^8.3.4",
+    "postcss-atroot": "^0.2.3",
+    "postcss-cli": "^8.3.1",
     "postcss-comment": "^2.0.0",
-    "postcss-each": "^0.10.0",
-    "postcss-functions": "3.0.0",
-    "postcss-import": "12.0.0",
-    "postcss-nested": "^4.2.3",
-    "postcss-scss": "^2.1.1",
-    "postcss-simple-vars": "5.0.2",
+    "postcss-each": "^1.0.0",
+    "postcss-extend-rule": "^3.0.0",
+    "postcss-focus-visible": "^5.0.0",
+    "postcss-functions": "^4.0.2",
+    "postcss-import": "^14.0.2",
+    "postcss-loader": "^4.2.0",
+    "postcss-map-get": "^0.3.0",
+    "postcss-nested": "^5.0.5",
+    "postcss-preset-env": "^6.7.0",
+    "postcss-property-lookup": "^3.0.0",
+    "postcss-scss": "^4.0.0",
     "postcss-strip-inline-comments": "^0.1.5",
-    "postcss-url": "8.0.0",
-    "precss": "^4.0.0",
-    "prettier": "^2.2.1",
-    "react-is": "^17.0.1",
-    "rollup-plugin-postcss": "^3.1.8",
+    "postcss-url": "^10.1.3",
+    "prettier": "^2.3.1",
+    "react-is": "^17.0.2",
+    "rollup-plugin-postcss": "^4.0.0",
     "rollup-plugin-url": "^3.0.1",
-    "sass-loader": "^10.1.1",
-    "size-limit": "^4.9.2",
+    "sass-loader": "^12.1.0",
```

Comment:
https://github.com/webpack-contrib/sass-loader/blob/master/CHANGELOG.md#1100-2021-02-05

Looks like as of version 11 we require webpack 5. Since this is a dev dependency it means only this project needs webpack 5, not every project that uses component library, right?